apollo-router/src/plugins/file_uploads/mod.rs

```diff
@@ -254,7 +254,7 @@ fn replace_value_at_path<'a>(
 
 // Removes value at path.
 fn remove_value_at_path<'a>(variables: &'a mut json_ext::Object, path: &'a [String]) {
-    let _ = get_value_at_path(variables, path).take();
+    let _ = get_value_at_path(variables, path).map(|v| v.take());
```

Comment:
Previously, it just replaced the reference to the value inside the option returned by `get_value_at_path`, so no mutation of `variables` occurred.
Now, if `variables` contain a value, it is correctly replaced with `null`.

Reply:
Maybe we can make this a bit more explicit by doing `*v = Value::Null`? it's a bit confusing to read it like this